File: Bindings~/java/src/main/java/ro/altom/alttester/Commands/UnityCommand/AltGetAllLoadedScenes.java
package ro.altom.alttester.Commands.UnityCommand;

import ro.altom.alttester.AltMessage;
import ro.altom.alttester.IMessageHandler;
import ro.altom.alttester.Commands.AltBaseCommand;

public class AltGetAllLoadedScenes extends AltBaseCommand {
    public AltGetAllLoadedScenes(IMessageHandler messageHandler) {
        super(messageHandler);
    }

    public String[] Execute() {
        AltMessage altMessage = new AltMessage();
        altMessage.setCommandName("getAllLoadedScenes");
        SendCommand(altMessage);
        return recvall(altMessage, String[].class);
    }
}

File: Assets/AltTester/AltDriver/Commands/UnityCommands/AltGetAllLoadedScenes.cs
using System.Collections.Generic;

namespace AltTester.AltDriver.Commands
{
    internal class AltGetAllLoadedScenes : AltBaseCommand
    {
        private readonly AltGetAllLoadedScenesParams cmdParams;
        public AltGetAllLoadedScenes(IDriverCommunication commHandler) : base(commHandler)
        {
            cmdParams = new AltGetAllLoadedScenesParams();
        }
        public List<string> Execute()
        {
            CommHandler.Send(cmdParams);
            return CommHandler.Recvall<List<string>>(cmdParams);
        }
    }
}